lib/components/gene/header.js

```diff
@@ -10,13 +10,14 @@ import Headline from '../text/headline'
 
 class Header extends React.Component {
   state: {
-    following: ?boolean
+    following: ?boolean,
   }
 
   static propTypes: Object = {
     gene: React.PropTypes.shape({
       name: React.PropTypes.string,
     }),
+    shortForm: React.PropTypes.bool
   }
```

Comment:
The header and sticky header parts of this view are complicated, so this "shortForm" probably won't survive longer than a PR or two, but for now, its a simple enough flag